packages/bee-q/src/components/notification/bq-notification.tsx

```diff
@@ -0,0 +1,169 @@
+
+  @State() private hasBtn = false;
+  @State() private isHidden = true;
+
+  // Public Property API
+  // ========================
+
+  /** Type of Notification */
+  @Prop({ reflect: true }) type: string;
+
+  /** Set property if you want Notification icon to be shown. */
+  @Prop({ reflect: true }) showIcon: boolean;
+
+  /** Set the subject color if you don't want to be black. Subject color will also apply to Icon color if there is one. */
+  @Prop({ reflect: true }) subjectColor: string;
+
+  /** Set property if to false if you want to hide Close icon */
+  @Prop({ reflect: true }) showClose: boolean;
+
+  // Prop lifecycle events
+  // =======================
+
+  // Events section
+  // Requires JSDocs for public API documentation
+  // ==============================================
+
+  // Component lifecycle events
+  // Ordered by their natural call order
+  // =====================================
+
+  // Listeners
+  // ==============
+
+  // Public methods API
+  // These methods are exposed on the host element.
+  // Always use two lines.
+  // Public Methods must be async.
+  // Requires JSDocs for public API documentation.
+  // ===============================================
+
+  /**
+   * Trigger function when you want to close Notification
+   */
+  @Method()
+  async hideNotification() {
+    this.closeNotification();
+  }
+
+  /**
+   * Trigger function when you want to show Notification
+   */
+  @Method()
+  async showNotification() {
+    this.setNotificationOpened();
+  }
+
+  // Local methods
+  // Internal business logic.
+  // These methods cannot be called from the host element.
+  // =======================================================
+
+  private getNotificationIconName = () => {
+    if (!this.showIcon) return '';
+
+    const type = this.type;
+    let icon = 'info';
+
+    if (type === 'success') {
+      icon = 'check-circle';
+    }
+
+    if (type === 'error') {
+      icon = 'x-circle';
+    }
+
+    return icon;
+  };
+
+  private handleSlotChange = () => {
+    this.hasBtn = hasSlotContent(this.buttonElem);
+  };
+
+  private closeNotification = () => {
+    this.isHidden = true;
+  };
+
+  private setNotificationOpened = () => {
+    this.isHidden = false;
+  };
+
+  // render() function
+  // Always the last one in the class.
+  // ===================================
+
+  render() {
+    const styles = {
+      ...(this.subjectColor && { color: getColorCSSVariable(this.subjectColor) }),
+    };
+
+    return (
+      <Host aria-hidden={this.isHidden} hidden={this.isHidden}>
+        <div
+          class="bg-white notification-shadow notification-radius inline-block w-auto px-5 py-4"
+          id="notification-main-holder"
+        >
+          {this.showIcon && (
+            <div class="mr-2 inline-block text-left align-top" id="notification-icon-holder">
+              <bq-icon name={this.getNotificationIconName()} color={this.subjectColor} size="18"></bq-icon>
```

Comment:
According to the design specs, the icon should be 24px (default size)

![CleanShot 2023-03-23 at 10 25 10](https://user-images.githubusercontent.com/328492/227145119-670d9f57-909a-4651-9449-3e34915c8600.png)